src/lib/numbers.ts

```diff
@@ -7,6 +7,8 @@ import { getPlusOrMinusSymbol } from "./utils";
 const MAX_EXCEEDING_THRESHOLD = "1000000000";
 const MIN_EXCEEDING_THRESHOLD = "0.01";
 
+export const BIG_ZERO = BigNumber.from(0);
```

Comment:
I think a better name would be `BN_ZERO` or you can use `ethers.constants.Zero`

Reply:
Good thought! Switched to BN_ZERO as the idea was to have a short constant name

Reply:
fyi, we use ZERO in subgraph

<img width="556" alt="image" src="https://github.com/gmx-io/gmx-interface/assets/144131435/eddf85e9-a6cf-45c3-940b-ede406eea783">

maybe we can do similar in UI, I'm not sure really though